### PR TITLE
Add jakarta to EL fat bucket

### LINKED ISSUE #12153: Update EL-3.0 FAT bucket to work with EE9 feature set
With the Jakarta namespace change we also need to update the `com.ibm.ws.el.3.0_fat` FAT bucket to run with the new EL-4.0 feature

*EDIT:* Ran into an issue with permissions locally. Locally we were hitting a java sec 2 issue which we had to add a permission for on the app. Whats weird is that the issue occurred for both el-3.0 AND with el-4.0. We never encountered this issue before on the previous builds with el-3.0 so we would need to investigate why this happened.

### PR DESCRIPTION
**Pull request by isaacrivriv**
fixes #12153 
- Added changes to include jakarta repeat
- Updated permissions for app
- Updated copyrights on files

### CODE REVIEW

**Comment by isaacrivriv** [comment]
#build

File: dev/com.ibm.ws.el.3.0_fat/test-applications/TestEL3.0.war/resources/META-INF/permissions.xml [hunk 1]
```diff
@@ -20,4 +20,9 @@
       <name>getClassLoader</name>  
    </permission>
 
+   <permission>  
+	  <class-name>java.lang.RuntimePermission</class-name>
+      <name>accessClassInPackage.com.sun.beans.editors</name>  
+   </permission>
+
```
Comment (pnicolucci) [inline dev/com.ibm.ws.el.3.0_fat/test-applications/TestEL3.0.war/resources/META-INF/permissions.xml:27]: What changed that requires this permission?

Comment (pnicolucci) [reply dev/com.ibm.ws.el.3.0_fat/test-applications/TestEL3.0.war/resources/META-INF/permissions.xml:27]: Could you also clean up the formatting of this class it's a bit messy?

**Comment by isaacrivriv** [comment]
#build

File: dev/com.ibm.ws.el.3.0_fat/bnd.bnd [hunk 2]
```diff
@@ -17,6 +17,10 @@ src: \
 
 fat.project: true
 
+tested.features:\
+	el-4.0,\
+    servlet-5.0,\
+    componenttest-2.0
```
Comment (pnicolucci) [inline dev/com.ibm.ws.el.3.0_fat/bnd.bnd:23]: Please fix the formatting here.

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#run-libby-bot

File: dev/com.ibm.ws.el.3.0_fat/test-applications/TestEL3.0.war/resources/META-INF/permissions.xml (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.el.3.0_fat/test-applications/TestEL3.0.war/resources/META-INF/permissions.xml:27]: Opened for investigation: https://github.com/OpenLiberty/open-liberty/issues/12284

**Pull request merged**